ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.less

```diff
@@ -1,23 +1,29 @@
 @{customElementNamespace}result-form {
   .customElement;
 
+  height: 100%;
   overflow: hidden;
 
   form {
     height: 100%;
     overflow: hidden;
+    position: relative;
   }
 
   .editor-properties {
     display: inline-block; // do not change back to block, fixes scrolling / repaint issues in IE10 / IE11
     width: 100%;
     overflow: auto;
-    max-height: ~'calc(100% - 2*@{minimumButtonSize})';
+    max-height: ~'calc(100% - @{minimumButtonSize})';
     padding: 10px 10px;
   }
 
   .editor-footer {
-    margin-top: 20px;
+    position: absolute;
```

Comment:
Originally this was setup to have it stick to the bottom on overflow.  With this change won't it always be stuck to the bottom, even if there's room directly underneath the editor properties?

Reply:
@andrewkfiedler Yes, with this change there is room directly underneath the editor properties that creates a gap between the buttons and the last field (Attributes), which gets bigger the further out the user sets their zoom. However, without setting this property and forcing the buttons to the bottom of the lightbox, the gap moves below the buttons. I thought forcing the buttons to the bottom was visually cleaner.